nova/tests/unit/api/openstack/compute/test_used_limits.py

```diff
@@ -129,8 +130,7 @@ def test_admin_can_fetch_limits_for_a_given_tenant_id(self):
             self.controller.index(fake_req, res)
             self.mock_can.assert_called_once_with(ul_policies.BASE_POLICY_NAME,
                                                   target)
-            mock_get_quotas.assert_called_once_with(self.fake_context,
-                tenant_id, usages=True)
+            self.assertEqual(2, mock_get_quotas.call_count)
```

Comment:
I think we should use the `expected_calls` pattern here to check the call args are the same, i.e. make these args from before a `mock.call()` and use `assert_has_calls()` with the two calls.